tools/src/scie_pants/ptex.py

```diff
@@ -45,5 +45,5 @@ def fetch_json(self, url: str, **headers: str) -> Any:
     def fetch_text(self, url: str, **headers: str) -> str:
         return self._fetch(url, stdout=subprocess.PIPE, **headers).stdout.decode()
 
-    def fetch_to_fp(self, url: str, fp: BinaryIO, **headers: str) -> None:
+    def fetch_to_fp(self, url: str, fp: IO, **headers: str) -> None:
```

Comment:
It feels incorrect to be potentially using text-IO for this, i.e. `BinaryIO` seems like the right type. What prompted this change?